features.md

```diff
@@ -140,6 +140,24 @@ The current version of the spec do not provide a way to enumerate the possible v
 }
 ```
 
+## <a name="featuresUnsafeAnnotationsInConfig" />Unsafe annotations in `config.json`
+
+**`unsafeAnnotationsInConfig`** (array of strings, OPTIONAL) contains values of [`annotations` property of `config.json`](config.md#annotations)
+that may change the behavior of the runtime.
```

Comment:
I still feel that this text should be more specific as to what "may change the behaviour of the runtime" means, but that's a minor nit.

Reply:
Any suggestions?